app/models/util/token.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+from app import db
+
+from app.models import Token, User
+
+def save_token(*, user_id: int, token: str) -> bool:
+    session = db.get_session()
+    user = session.query(User).filter(User.id == user_id).one_or_none()
+
+    # If user does not exist, do not save token and return False as error
+    if not user:
+        return False
+
+    session.add(Token(token_str=token, time_created=datetime.utcnow(), user=user))
```

Comment:
"SQLite DateTime type only accepts Python datetime and date objects as input."
Suggestion: set a default for time_created in the model so it doesn't have to be passed.

Reply:
Fixed! I done fucked up

Reply:
don't see commit

Reply:
Let me check, running on ipython:
```
In [1]: from datetime import datetime

In [2]: datetime.utcnow()
Out[2]: datetime.datetime(2019, 1, 11, 20, 37, 26, 850597)

In [3]: type(datetime.utcnow())
Out[3]: datetime.datetime
```

and I remember stumbling into this issue and fixing it so idk what's going on